src/Components/Publishing/Fixtures/Components.ts

```diff
@@ -5,7 +5,7 @@ export const Artworks = [
     type: "artwork",
     id: "589a6291275b2410d1beb6a5",
     slug: "fernando-botero-nude-on-the-beach",
-    date: "2000",
+    date: "",
     title: "Nude on the Beach",
```

Comment:
Curious why you wanted to remove the date here?

Reply:
The fixture is meant to represent an artwork with missing info